src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Altinn.Platform.Authorization.csproj

```diff
@@ -22,6 +25,10 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\Authorization.Interface\Authorization.Interface.csproj" />
```

Comment:
Should we add it as a packagereference actually using the nuget package instead of referencing the project?

Reply:
Yes, but i think we could publish nuget when we know that it works as expected with bridge. So that we don't end up publishing nuget package very often. So I suggest that we publish once it's stable